runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -875,6 +896,40 @@ private void removeTzhaarTimer()
 		}
 	}
 
+	private void checkTeleport(VarPlayer varPlayer)
+	{
+		final GameTimer teleport;
+		final int time;
+		switch (varPlayer)
+		{
+			case LAST_HOME_TELEPORT:
+				teleport = HOME_TELEPORT;
+				time = 30;
```

Comment:
The length of these timers can be gotten from the timer itself, eg. `teleport.getDuration().getSeconds()`, which can then be fed back into the expiry instant via `Instant.ofEpochSecond(lastTeleportSeconds).plus(teleportCooldownSeconds, ChronoUnit.SECONDS)`. That avoids hard-coding the duration, when it's already present in the enum definition.